spec/support/patch_jsonapi_matcher_clash.rb

```diff
@@ -0,0 +1,29 @@
+# https://github.com/jsonapi-rb/jsonapi-rspec/issues/4
+#
+# jsonapi-rspec replaces RSpec's BuiltIn have_attributes matcher, which
+# is useful in language_spec.rb for validating the use of validators via
+# +an_object_having_attributes+. While jsonapi-rspec is extremely naughty
+# and a bit self-important here, without patching that gem our options are
+# limited, because we're using its patched +have_attributes+ in many
+# API-related specs.
+#
+# Since we only use +an_object_having_attributes+, and that isn't in use
+# in the API specs replace that here. And consider either dropping or
+# patching jsonapi-rspec, which is at the time of writing 2 years without
```

Comment:
Do you have a sense of how much work it would be to remove `jsonapi-rspec`?

Reply:
Had a brief look; it wasn't the case that we could just pull a few matchers in and remove the gem. Looked non-trivial. TBH we don't need this commit any more as we went with `have_field` in the end but rebasing it out was also non-trivial for reasons I don't remember.